### DISCUSSION TITLE
How to perform hyperparameter optimization with mmpretrain?

### DISCUSSION THREAD

**Discussion by tjwielem** [open-mmlab/mmclassification#1916] [Q&A]
I'm wondering if there is a way to perform hyperparameter optimization with mmpretrain. I'd like to optimize some parameters such as batch size, optimizer type, initial learning rate and a few others. There seem to be many frameworks, like RayTune and NNI, for using popular hyperparameter optimizers, however I can't seem to find a method to do this with mmpretrain. I came across SiaTune which claims to do it, however I am hesitant on using it as it hasn't been updated in over a year. Any help would be appreciated.
